### PR TITLE
Add note to README about not using fastrtps

### PR DESCRIPTION
**Pull request by jacquelinekay**

### CODE REVIEW

**Comment by gerkey** [comment]
Can we point to an existing (FastRTPS) ticket that is tracking the what's missing for the bridge?

File: README.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ The following ROS 1 packages are required to build and use the bridge:
 * `std_msgs`
 * as well as the Python package `rospkg`
 
+**WARNING**: The ROS 1 bridge currently does not work if ROS 2 is built with the FastRTPS middleware.
+If you installed ROS 2 from source, make sure you installed it with another supported middleware (Opensplice or Connext).
```
Comment (dirk-thomas) [inline README.md]: If you installed ROS 2 from source, make sure you **built** it with ... ?

**Comment by jacquelinekay** [comment]
https://github.com/eProsima/ROS-RMW-Fast-RTPS-cpp/issues/14 is the original ticket for the missing rmw functions that are called by the rclcpp functions used by the bridge to collect existing topic names. There are references to other issues there (e.g. #22 was specific to the ROS 1 bridge but was closed as a duplicate of the first issue)

**Comment by jacquelinekay** [comment]
Addressed comments.

**Comment by gerkey** [comment]
+1

**Comment by kartikmohta** [comment]
I think this works now since the functions have been implemented in Fast RTPS (https://github.com/eProsima/ROS-RMW-Fast-RTPS-cpp/pull/56), so the note can be removed from the README

**Comment by dirk-thomas** [comment]
@kartikmohta Thank you for pointing this out. I have reverted the commit in bd55d29aab567607337b3de9b4685e0ec10536de.

**Pull request merged**